packages/tools/src/tools/segmentation/RectangleROIStartEndThresholdTool.ts

```diff
@@ -276,6 +350,7 @@ class RectangleROIStartEndThresholdTool extends RectangleROITool {
     // Since we are extending the RectangleROI class, we need to
     // bring the logic for handle to some cachedStats calculation
     this._computeProjectionPoints(annotation, imageVolume);
+    this._computePointsInsideVolume(annotation, imageVolume, enabledElement);
```

Comment:
Could you consider placing it under a configuration? This way, it will add extra steps only when necessary for the use case.